src/Assignments/ProblemIframe.tsx

```diff
@@ -222,7 +228,10 @@ export const ProblemIframe: React.FC<ProblemIframeProps> = ({
 
                 if(_.isNil(iframeRef?.current)) {
                     logger.error('Hijacker: Could not find the iframe ref');
-                    setError('An error occurred');
+                    setAlert({
+                        variant: 'danger',
+                        message: 'There was an error with the problem iframe.'
```

Comment:
This error message is very developery
A student is likely going to be offput by `iframe`

A better error message might look like:
`There was an error rendering the problem.`